src/kili/services/plugins/tools.py

```diff
@@ -0,0 +1,13 @@
+"""
```

Comment:
Why calling this file tools and not helpers?

Reply:
I wanted to keep same wording as the export service